### ISSUE TITLE
Add a standard right-click/context menu? 

### ISSUE THREAD

**Issue by Ryder17z** [kovidgoyal/kitty#7632]
**Is your feature request related to a problem? Please describe.**
Kitty seems good, but not being able to easily copy/cut/paste is a dealbreaker for me.
Having a context menu really helps as keyboard shortcuts is a mess if you get used to how GUI applications work as they generally conflict with how terminals and etc cli works.

**Describe the solution you'd like**
A similar right-click/context menu as to what is provided by for example Konsole

**Describe alternatives you've considered**
Going back to Konsole

**Additional context**

**Comment by kovidgoyal** [comment]
No, there is going to be no context menu. kitty is for terminal poer
users. No power user is going to use a context menu for copy/paste, it's
too inefficient. If you want to copy/paste using GUI shortcuts you can do so in kitty with

map ctrl+c copy_or_interrupt
map ctrl+v paste

**Comment by Ryder17z** [comment]
Has it ever occurred to you that power users might be migrating away from windows?

**Comment by kovidgoyal** [comment]
And?

**Comment by Ryder17z** [comment]
Edit: I just noticed the mouse controls are pretty easy to remember. It'll do the trick for me.

**Comment by fakstory** [comment]
This thread came up in search but didn’t quite solve the practical workflow issue for me, so I’m sharing the configuration that worked — in case it helps others looking for a simple, coffee-friendly right-click paste setup.

For some workflows (especially casual ones ☕), a single right-click paste from the clipboard can be more ergonomic than reaching for Ctrl+Shift+V. That’s not about replacing keyboard shortcuts — just adapting behavior to context.

Kitty’s minimalist philosophy is clear, and this configuration keeps that approach while enabling a more mouse-friendly setup:

If you are a power user who also drinks coffee, you might appreciate this:

```
# 1. Clear the default right-click behavior
mouse_map right press ungrabbed no_op

# 2. Set right-click to paste from clipboard (coffee mode ☕)
mouse_map right click ungrabbed paste_from_clipboard

# Shift+Right click to extend the current selection
mouse_map shift+right press ungrabbed mouse_selection extend

# Optional: automatically copy selection (I just use middle click when drinking coffee, I want primary and clipboard to be separate)
# copy_on_select yes

# Ctrl+C: copy if selection exists, otherwise interrupt
map ctrl+c copy_or_interrupt

# Ctrl+Shift+C: always send SIGINT
map ctrl+shift+c send_text all \x03

# Optional: paste shortcuts
map ctrl+v paste_from_clipboard
map ctrl+shift+v paste_from_clipboard

```
This keeps classic terminal behavior intact while allowing both keyboard-first and ergonomic mouse-based workflows.

**Comment by fakstory** [comment]
TL;DR, if you are here first for the right+click menu, save yourself precious time, and switch to an other terminal now. or, bookmark this issue for your next "wt..." moment using kitty.

When ever you'll ssh on a fresh server, you'll hit a wall because the creator of kitty create a new, never use xterm-kitty instead of just use - > xterm-256color
```
sudo nano /mnt/etc/fstab
Error opening terminal: xterm-kitty.
```

That error is a classic case of a "handshake" failure between your terminal (Kitty) and the remote or environment-shifted system you're trying to work in.

Basically, the system you're accessing doesn't recognize xterm-kitty as a valid terminal type, so it doesn't know how to render the text interface for nano. This often happens when you've ssh-ed into a server or chroot-ed into a mount point that lacks the Kitty terminfo files.

Here are the quickest ways to fix it:
1. The "Quick & Dirty" Fix (Most Reliable)

Tell the system to treat your session as a standard xterm-256color, which almost every Linux system understands. Run this command before trying to open the file:
```
Bash

export TERM=xterm-256color
```

Then run your command again: `sudo nano /mnt/etc/fstab`

2. The "Correct" Fix (If using SSH)

If you are SSH-ing into this machine frequently, Kitty has a built-in tool to "teach" the remote server what Kitty is. On your local machine (not the server), run:
```
Bash

kitten ssh-copy-terminfo user@remote-host
```

This clones the necessary definition files to the remote server so it recognizes xterm-kitty natively.

3. The "Emergency" Fix

If xterm-256color still gives you grief, you can drop back to the most basic, no-frills setting:
```
Bash

export TERM=vt100
```

Note: You might lose fancy colors or some keyboard shortcuts, but nano will definitely open.

This felt like a non-powerful feature.
